src/schedule.py

```diff
@@ -148,6 +163,43 @@ def schedule_revoke_event(
     )
 
 
+def schedule_group_revoke_event(
+    schedule_client: EventBridgeSchedulerClient,
+    permission_duration: timedelta,
+    approver: entities.slack.User,
+    requester: entities.slack.User,
+    group_assignment: sso.GroupAssignment,
+) -> scheduler_type_defs.CreateScheduleOutputTypeDef:
+    logger.info("Scheduling revoke event")
+    schedule_name = f"{cfg.revoker_function_name}" + datetime.now().strftime("%Y-%m-%d-%H-%M-%S")
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The naive datetime objects are treated by many datetime methods as local times, it is preferred to use aware datetimes to represent times in UTC. The recommended way to create an aware datetime object representing a specific timestamp in UTC is by passing `tzinfo` as an argument to the method.

[Learn more](https://docs.python.org/3/library/datetime.html#aware-and-naive-objects)